<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>586</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Comparator</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="path1Label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Path 1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="path1LineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>25</y>
      <width>501</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>/home/sasha/other/matching/3.ts</string>
    </property>
   </widget>
   <widget class="QLabel" name="path2Label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>65</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Path 2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="path2LineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>60</y>
      <width>501</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>/home/sasha/other/matching/3-ctl.ts</string>
    </property>
   </widget>
   <widget class="QLabel" name="shiftLabel">
    <property name="geometry">
     <rect>
      <x>27</x>
      <y>155</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>shift</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="shiftLineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>150</y>
      <width>61</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>-91</string>
    </property>
   </widget>
   <widget class="QPushButton" name="syncButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>sync</string>
    </property>
   </widget>
   <widget class="QLabel" name="corrLabel">
    <property name="geometry">
     <rect>
      <x>31</x>
      <y>185</y>
      <width>31</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>corr</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="corrLineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>180</y>
      <width>81</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="diffButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>150</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>diff</string>
    </property>
   </widget>
   <widget class="QLabel" name="jsonLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>105</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>JSON</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="jsonLineEdit">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>100</y>
      <width>501</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="endLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>255</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>end</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="beginLineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>215</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>400</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="endLineEdit">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>250</y>
      <width>121</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>800</string>
    </property>
   </widget>
   <widget class="QLabel" name="beginLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>51</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>begin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="incButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>inc</string>
    </property>
   </widget>
   <widget class="QLabel" name="syncSkipLabel">
    <property name="geometry">
     <rect>
      <x>367</x>
      <y>155</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>sync_skip</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="syncSkipLineEdit">
    <property name="geometry">
     <rect>
      <x>441</x>
      <y>150</y>
      <width>60</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
